data/json/furniture.json

```diff
@@ -3381,5 +3381,207 @@
       "ter_set": "t_floor",
       "items": [ { "item": "pillow", "count": 19 }, { "item": "blanket", "count": 3 } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_file_cabinet",
+    "name": "filing cabinet",
+    "description": "A set of drawers in a sturdy metal cabinet, used to hold files.  It can be locked to protect important information.",
+    "symbol": "}",
+    "color": "dark_gray",
+    "move_cost_mod": 2,
+    "required_str": 8,
+    "flags": [ "PLACE_ITEM", "TRANSPARENT", "CONTAINER", "BLOCKSDOOR", "MOUNTABLE" ],
+    "deconstruct": {
+      "items": [
+        { "item": "sheet_metal", "count": [ 2, 6 ] },
+        { "item": "scrap", "count": [ 2, 6 ] },
+        { "item": "clockworks", "count": [ 0, 2 ] }
+      ]
+    },
+    "max_volume": 4000,
+    "bash": {
+      "str_min": 8,
+      "str_max": 30,
+      "sound": "metal screeching!",
+      "sound_fail": "clang!",
+      "items": [
+        { "item": "scrap", "count": [ 0, 6 ] },
+        { "item": "sheet_metal", "count": [ 0, 4 ] }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_wardrobe",
+    "name": "wardrobe",
+    "description": "A tall piece of furniture - basically a freestanding closet.",
+    "symbol": "{",
+    "color": "i_brown",
+    "move_cost_mod": -1,
+    "required_str": 14,
+    "flags": [ "CONTAINER", "FLAMMABLE", "PLACE_ITEM", "ORGANIC", "BLOCKSDOOR", "MOUNTABLE" ],
+    "deconstruct": { "items": [ { "item": "2x4", "count": 12 }, { "item": "nail", "charges": [ 8, 12 ] } ] },
+    "max_volume": 16000,
+    "bash": {
+      "str_min": 12,
+      "str_max": 40,
+      "sound": "smash!",
+      "sound_fail": "whump.",
+      "items": [ { "item": "2x4", "count": [ 4, 8 ] }, { "item": "nail", "charges": [ 6, 12 ] }, { "item": "splinter", "count": 1 } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_piano",
+    "name": "piano",
+    "description": "The ol' ebony and ivory.",
+    "symbol": "P",
+    "color": "i_black",
+    "move_cost_mod": 2,
```

Comment:
It is passable? And if so, absolutely no penalties, like moving over grass?

Reply:
Move cost mod 2 is the same as other flat-topped things like tables and countertops. I think the assumption was it was a close-top grand piano that you could vault over.  That said, I think it should be higher... it sits higher off the ground than those things and may be open and hard to move overtop.